### PR TITLE
Traefik for prod

### PR DESCRIPTION
**Pull request by raulsperoni**
Integré el servicio de traefik al docker-compose.prod.yml. El dashboard que expone está bueno y podemos activar ahí mismo access logs, etc. Actualicé el README también.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,4 +158,35 @@ DB_PASS=<VALUE>
...
 ```
+
+For every service you want exposed through the reverse proxy you should add this to the service block in `docker-compose.yml` file and change `SERVICE_NAME`:
+```
+depends_on:
+  - traefik
+networks:
+  - proxy
+labels:
+  - traefik.http.routers.whoami.rule=Host(`SERVICE_NAME.${TRAEFIK_DOMAIN}`)
```
Comment (bryant1410) [inline README.md:172]: Does the `.env` file get loaded when reading the Docker Compose config file?

File: docker-compose.prod.yml [hunk 2]
```diff
@@ -16,3 +19,40 @@ services:
...
+          - "--certificatesResolvers.le.acme.storage=acme.json"
+          - "--certificatesResolvers.le.acme.tlsChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge.entryPoint=web"
+          - "--log.level=INFO"
+      restart: always
+      ports:
+          - 80:80
+          - 443:443
+          - 8080:8080
```
Comment (bryant1410) [inline docker-compose.prod.yml:41]: What is this port for?

File: docker-compose.prod.yml [hunk 3]
```diff
@@ -16,3 +19,40 @@ services:
...
+      networks:
+          - proxy
+      volumes:
+          - /var/run/docker.sock:/var/run/docker.sock
```
Comment (bryant1410) [inline docker-compose.prod.yml:45]: Can this one be read only?

File: docker-compose.prod.yml [hunk 4]
```diff
@@ -1,4 +1,7 @@
 version: "3.8"
+networks:
+  proxy:
```
Comment (bryant1410) [inline docker-compose.prod.yml:3]: Maybe we can remove the `frontend` network.

**Review by bryant1410 — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (raulsperoni) [reply README.md:172]: Yes...

File: docker-compose.prod.yml (hunk 2 above)
Comment (raulsperoni) [reply docker-compose.prod.yml:41]: Traefik dashboard, it's useful while settings things up, we can remove it after

File: docker-compose.prod.yml (hunk 3 above)
Comment (raulsperoni) [reply docker-compose.prod.yml:45]: I don't know. I found this but haven't tested it. https://chriswiegman.com/2019/11/protecting-your-docker-socket-with-traefik-2/

File: docker-compose.prod.yml (hunk 4 above)
Comment (raulsperoni) [reply docker-compose.prod.yml:3]: Yes definitely. I didn't want to remove anything but should be the same

**Pull request merged**